target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -10,10 +10,20 @@ board=$(mvebu_board_name)
 case "$board" in
 armada-xp-linksys-mamba)
         # Set one execution of the fan script during every start (waiting for cron)
-        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
+        if grep -q fan_ctrl.sh /etc/rc.local
+                then
+                        exit 0
+                else
+                        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
+        fi                        
         # Set fan script execution in crontab
-        echo "# mamba fan script runs every 5 minutes" >> /etc/crontabs/root
-        echo "*/5 * * * * /sbin/fan_ctrl.sh" >> /etc/crontabs/root
+        if grep -q fan_ctrl.sh /etc/crontabs/root
```

Comment:
Use `grep -s -q ...` to suppress errors in case `/etc/crontabs/root` does not exist yet.